homeassistant/components/webhook/__init__.py

```diff
@@ -90,16 +104,18 @@ async def async_handle_webhook(
     """Handle a webhook."""
     handlers: dict[str, dict[str, Any]] = hass.data.setdefault(DOMAIN, {})
 
+    content_stream: StreamReader | MockStreamReader
+    if isinstance(request, MockRequest):
+        received_from = request.mock_source
+        content_stream = request.content
+        method_name = request.method
+    else:
+        received_from = request.remote
+        content_stream = request.content
+        method_name = request.method
```

Comment:
The last two statements are the same in each block so they can be outside the if/else

Reply:
The original code also had content_stream twice. I believe this was intentional to avoid two errors in mypy.

```
homeassistant/components/webhook/__init__.py:108: error: Argument 1 to "__get__" of "reify" has incompatible type "Union[Request, MockRequest]"; expected "_TSelf[StreamReader]"  [arg-type]
homeassistant/components/webhook/__init__.py:109: error: Argument 1 to "__get__" of "reify" has incompatible type "Union[Request, MockRequest]"; expected "_TSelf[str]"  [arg-type]
Found 2 errors in 1 file (checked 1 source file)
```

Reply:
Ok. Strange, but fair enough.